corehq/apps/importer/tasks.py

```diff
@@ -104,8 +104,11 @@ def _submit_caseblocks(caseblocks):
             if not any(fields_to_update.values()):
                 # if the row was blank, just skip it, no errors
                 continue
-        except importer_util.InvalidDateException:
-            errors.add(ImportErrors.InvalidDate, i + 1)
+        except importer_util.InvalidDateException as e:
+            errors.add(ImportErrors.InvalidDate, i + 1, e.column)
```

Comment:
Is this being `i + 1` affected by whether or not the user selected "This file contains column headers"? Should it be?

(Realize you're following the same pattern already in this file, just curious.)